### DISCUSSION TITLE
hediye sandığı saati yanlışmı çalışıyor

### DISCUSSION THREAD

**Discussion by ozgurizmir** [IBAX-io/go-ibax#589] [Soru Cevap - Türkçe] [answered]
Saat hediyesi yanlışmı çalışıyor ?

**Comment by Yusufknk** [comment, marked as answer]
Hayır, ilk açılıştan 24 saat sonra tekrar açabilirsiniz. Hergün açtığınızda beklenen zaman miktarı da azalır.

**Reply by Azizatli21** [reply]
Merhaba otomatik olarak mi aciliyor

**Comment by Eymenedip** [comment]
Hediye sandığı kaç puan veriyor herkeste aynimi

**Reply by mrtgezgin** [reply]
> Hediye sandığı kaç puan veriyor herkeste aynimi

Hayır herkesin farklı olabiliyor.

**Reply by Huseyincalik67** [reply]
Tamamdır teşekkürler

**Comment by Azizatli21** [comment]
Nasıl herkese farklı mi geliyor
